Client/Frontend/Widgets/PhotonActionSheetProtocol.swift

```diff
@@ -190,6 +191,21 @@ extension PhotonActionSheetProtocol {
             }
         }
 
+        let removeTopSitesPin = PhotonActionSheetItem(title: Strings.RemovePinTopsiteActionTitle, iconString: "action_unpin") { action in
+            guard let url = tab.url?.displayURL,
+                let sql = self.profile.history as? SQLiteHistory else { return }
+            let absoluteString = url.absoluteString
```

Comment:
Lets not create this here. We can just pass `url.absoluteString` to the line below